### PR TITLE
Updated README.md

### LINKED ISSUE #3: Update Documentation
Update the format of documentation according to [this](https://github.com/metakgp/README.md)

### PR DESCRIPTION
**Pull request by Bikram-ghuku**
Fixes: #3

### CODE REVIEW

**Comment by proffapt** [comment]
Request for review once done.

**Comment by Bikram-ghuku** [comment]
> Request for review once done.

Review pls

File: README.md [hunk 1]
```diff
@@ -1,100 +1,103 @@
...
-![Cover](/screenshots/cover_new.png)
 
-# ERP Auto Login - IITKGP
+<!-- TABLE OF CONTENTS -->
+<details>
+<summary>Table of Contents</summary>
 
-Our default branch `master` is the source code to the recent stable release. Want to test beta versions? Head to the [`develop` branch](https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/tree/develop) to view the latest updates and features before the official release.
+- [About The Project](#about-the-project)
+  - [Supports](#supports)
```
Comment (proffapt) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,100 +1,103 @@
...
-  <a href="https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/issues/new">Request feature</a>
-  ·
-  <a href="https://addons.mozilla.org/en-US/firefox/addon/erp-auto-login-iitkgp/reviews">Feedback</a>
-</p>
+  <p align="center">
+  <!-- UPDATE -->
+    Minimial yet powerful and customizable autologin/autofill extension, for IIT Kharagpur enrolled students.
+    <br />
+    <a href="https://UPDATE.metakgp.org">Website</a>
+    ·
```
Comment (proffapt) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,100 +1,103 @@
...
+<div id="supports"></div>
 
-  > For android you can use extensions on the [Nightly](https://play.google.com/store/apps/details?id=org.mozilla.fenix&hl=en&gl=US) version of Firefox. [follow this article](https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/) to set it up.
+### Supports:
+<!-- UPDATE -->
+1. Shells
+    * `bash`
+    * `zsh`
+2. OS(s)
+    * any `*nix`[`GNU+Linux` and `Unix`]
```
Comment (proffapt) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,100 +1,103 @@
...
-  - [Click here](https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/releases) to download the latest release `.xpi` for firefox and `.crx` for chrome
-  - Load the extension in `Firefox` as:
-    1. Open Add-on Manager from options (or) by pressing <kbd>Ctrl/Cmd+Shift+A</kbd> on Widows/Mac
-    2. Click the gear icon ⚙️ in the upper-right area of the Add-on Manager Extensions panel.
-    3. Select Install Add-on from file... from the menu, then find and select the dowwloaded (.xpi) file.
-  - Load the extension in `Chrome` as:
-    1. Goto `chrome://extensions/` in your chrome browser
-    2. Enable developer mode by turning the switch on from the top right corner of the chrome browser
-    3. Click on load unpacked and find and select the dowwloaded (.crx) file.
+## Getting Started
```
Comment (proffapt) [inline README.md]: Rename this section to configuring the development environment or something similar and smaller. or just Building from source.

**Review by proffapt — CHANGES_REQUESTED** [review]
- [x] Add LICENSE from the readme.md repo as well
- [ ] Take inspiration from [ferp](https://github.com/proffapt/fERP)      
      image
      This will be the real installation part with those "add to firefox" "add to chrome" images as welll.

**Comment by Bikram-ghuku** [comment]
The links to the chrome and firefox are the links to the old versions...

File: README.md [hunk 5]
```diff
@@ -106,71 +107,121 @@ Our default branch `master` is the source code to the recent stable release. Wan
...
-
-## Previous releases
+- #### Deployment
 
-You can find all our [previous releases](https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/releases) on releases page of the github repo or directly from [AMO Website](https://addons.mozilla.org/en-US/firefox/addon/erp-auto-login-iitkgp/versions/)
-
-## Todos
+  1. Clone the repository
+     ```bash
+     git clone https://github.com/siddsarkar/iitkgp-erp-auto-login-extension.git
```
Comment (proffapt) [inline README.md]:
```suggestion
     git clone https://github.com/metakgp/iitkgp-erp-auto-login.git
```

Update it in all other places too.

File: README.md [hunk 6]
```diff
@@ -1,100 +1,101 @@
...
-  ·
-  <a href="https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/issues/new">Request feature</a>
-  ·
-  <a href="https://addons.mozilla.org/en-US/firefox/addon/erp-auto-login-iitkgp/reviews">Feedback</a>
-</p>
+  <p align="center">
+  <!-- UPDATE -->
+    Minimial yet powerful and customizable autologin/autofill extension, for IIT Kharagpur enrolled students.
+    <br />
+    <a href="https://github.com/proffapt/iitkgp-erp-auto-login/issues">Request Feature / Report Bug</a>
```
Comment (proffapt) [inline README.md]:
```suggestion
    <a href="https://github.com/metakgp/iitkgp-erp-auto-login/issues">Request Feature / Report Bug</a>
```

File: README.md [hunk 7]
```diff
@@ -106,71 +107,121 @@ Our default branch `master` is the source code to the recent stable release. Wan
...
+</p>
 
-See [the Releases section of our GitHub project](https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/releases) for changelogs for each release version of this extension.
+### Maintainer(s)
 
-## Thanks
+The currently active maintainer(s) of this project.
 
-Thanks to [vs code](https://code.visualstudio.com/) and [web-ext](https://www.npmjs.com/package/web-ext) for providing the environment that allowed us to develop and test in real browsers!
+<!-- UPDATE -->
```
Comment (proffapt) [inline README.md]:
```suggestion

```

File: README.md [hunk 8]
```diff
@@ -106,71 +107,121 @@ Our default branch `master` is the source code to the recent stable release. Wan
...
-You can find all our [previous releases](https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/releases) on releases page of the github repo or directly from [AMO Website](https://addons.mozilla.org/en-US/firefox/addon/erp-auto-login-iitkgp/versions/)
-
-## Todos
+  1. Clone the repository
+     ```bash
+     git clone https://github.com/siddsarkar/iitkgp-erp-auto-login-extension.git
+     ```
+  2. Cd into the repository
+     ```bash
+     cd iitkgp-erp-auto-login-extension
```
Comment (proffapt) [inline README.md]:
```suggestion
     cd iitkgp-erp-auto-login
```

File: README.md [hunk 9]
```diff
@@ -1,100 +1,101 @@
...
+  ```
+* web-ext
+  ```sh
+  npm install --global web-ext
+  ```
 
-- Build from source (for development)
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+### Building From Source
```
Comment (proffapt) [inline README.md]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -1,100 +1,101 @@
...
+
+- #### Development
```
Comment (proffapt) [inline README.md]:
```suggestion
### Development
```

File: README.md [hunk 11]
```diff
@@ -106,71 +107,121 @@ Our default branch `master` is the source code to the recent stable release. Wan
...
      web-ext run -t chromium # (or) run the extension for chrome
      ```
   7. Check that the extension is loaded by opening the extensions page in Firefox `about:addons` or in Chrome `chrome://extensions/`.
 
-## Bugs and feature requests
-
-Have a bug or a feature request? Please first search for existing and closed issues. If your problem or idea is not addressed yet, [please open a new issue](https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/issues/new).
-
-## Previous releases
+- #### Deployment
```
Comment (proffapt) [inline README.md]:
```suggestion
### Deployment
```

File: README.md [hunk 12]
```diff
@@ -1,100 +1,101 @@
...
+ERP Auto Login is a chrome/ firefox extension which can be added to the browser. The extension stores the users confidentials like username, password and security questions and fills them up automatically whenever the user opens the ERP. It also sends the otp to the email, the user only need to fill up the OTP and can login.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Get it from Firefox/ Chrome store
 
-  > For android you can use extensions on the [Nightly](https://play.google.com/store/apps/details?id=org.mozilla.fenix&hl=en&gl=US) version of Firefox. [follow this article](https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/) to set it up.
+<p align="Center">
+  <a href="https://addons.mozilla.org/en-US/firefox/addon/erp-auto-login-iitkgp/">
+    <img src="https://blog.mozilla.org/addons/files/2020/04/get-the-addon-fx-apr-2020.svg" alt="ERP Auto Login - IITKGP logo" height="58">
```
Comment (proffapt) [inline README.md]:
```suggestion
    <img src="https://blog.mozilla.org/addons/files/2020/04/get-the-addon-fx-apr-2020.svg" alt="ERP Auto Login - IIT KGP logo" height="58">
```

File: README.md [hunk 13]
```diff
@@ -1,100 +1,101 @@
...
+  </a>
+  <a href="https://chrome.google.com/webstore/detail/erp-auto-login-iitkgp/hianmedhblomknonbbmalncjjffdligl">
+    <img src="https://storage.googleapis.com/web-dev-uploads/image/WlD8wC6g8khYWPJUsQceQkhXSlv1/UV4C4ybeBTsZt43U4xis.png" alt="ERP Auto Login - IITKGP logo">
```
Comment (proffapt) [inline README.md]:
```suggestion
    <img src="https://storage.googleapis.com/web-dev-uploads/image/WlD8wC6g8khYWPJUsQceQkhXSlv1/UV4C4ybeBTsZt43U4xis.png" alt="ERP Auto Login - IIT KGP logo">
```

File: README.md [hunk 14]
```diff
@@ -1,100 +1,101 @@
...
+<!-- ABOUT THE PROJECT -->
+## About The Project
+<!-- UPDATE -->
+<div align="center">
+  <a href="https://github.com/metakgp/iitkgp-erp-auto-login">
+    <img width="80%" alt="image" src="./screenshots/cover_new.png">
+  </a>
+</div>
+
+ERP Auto Login is a chrome/ firefox extension which can be added to the browser. The extension stores the users confidentials like username, password and security questions and fills them up automatically whenever the user opens the ERP. It also sends the otp to the email, the user only need to fill up the OTP and can login.
```
Comment (proffapt) [inline README.md]:
```suggestion
ERP Auto Login is a browser extension supporting both chrome and firefox. The extension stores the users confidentials like username, password and security questions and fills them up automatically whenever the user opens the ERP. It also sends the OTP to the email, the user only need to fill up the OTP and can login.
```

File: README.md [hunk 15]
```diff
@@ -1,100 +1,101 @@
...
-Our default branch `master` is the source code to the recent stable release. Want to test beta versions? Head to the [`develop` branch](https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/tree/develop) to view the latest updates and features before the official release.
+- [About The Project](#about-the-project)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Contact](#contact)
+  - [Maintainer(s)](#maintainers)
+  - [creators(s)](#creators)
+- [Additional documentation](#additional-documentation)
```
Comment (proffapt) [inline README.md]: Update this, doesn't look like this is updated :)

File: README.md [hunk 16]
```diff
@@ -1,100 +1,101 @@
...
-  Minimial yet powerful and customizable autologin/autofill extension, for IIT Kharagpur enrolled students.
-  <br>
-  <a href="https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/issues/new">Report a bug</a>
-  ·
-  <a href="https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/issues/new">Request feature</a>
-  ·
-  <a href="https://addons.mozilla.org/en-US/firefox/addon/erp-auto-login-iitkgp/reviews">Feedback</a>
-</p>
+  <p align="center">
+  <!-- UPDATE -->
```
Comment (proffapt) [inline README.md]:
```suggestion
```

File: README.md [hunk 17]
```diff
@@ -1,100 +1,101 @@
...
+<br />
+<!-- UPDATE -->
+<div align="center">
+  <a href="https://github.com/metakgp/iitkgp-erp-auto-login">
+     <img width="140" alt="image" src="./src/assets/images/ext_icon.png">
   </a>
-</p>
 
-<h3 align="center">ERP Auto Login - IITKGP</h3>
+  <h3 align="center">ERP Auto Login - IITKGP</h3>
```
Comment (proffapt) [inline README.md]:
```suggestion
  <h3 align="center">ERP Auto Login - IIT KGP</h3>
```

**Review by proffapt — CHANGES_REQUESTED** [review]
- [x] Add manual installation steps, as shown in [fERP](https://github.com/proffpat/ferp)
- [x] Move those store installation icons below the text (below the extension logo)
- [ ] Edit that main image and mask the github URL

**Review by proffapt — APPROVED** [review]

**Pull request merged**